CONTRIBUTING.md

```diff
@@ -70,27 +70,97 @@ Sample curl request to invoke Ratify endpoint:
 curl -X POST http://127.0.0.1:6001/ratify/gatekeeper/v1/verify -H "Content-Type: application/json" -d '{"apiVersion":"externaldata.gatekeeper.sh/v1alpha1","kind":"ProviderRequest","request":{"keys":["localhost:5000/net-monitor:v1"]}}'
 ```
 ### Test local changes in the k8s cluster scenario
-There are some changes that should be validated in a cluster scenario.   
-Follow the steps below to build and deploy a Ratify image with your private changes:   
-1. build an image with your local changes
+There are some changes that should be validated in a cluster scenario.
+Follow the steps below to build and deploy a Ratify image with your private changes:
+
+#### build an image with your local changes
 ```
 docker build -f httpserver/Dockerfile -t yourregistry/deislabs/ratify:yourtag .
 ```
-2. [Authenticate](https://docs.docker.com/engine/reference/commandline/login/#usage) with your registry,  and push the newly built image
+
+#### [Authenticate](https://docs.docker.com/engine/reference/commandline/login/#usage) with your registry,  and push the newly built image
 ```
 docker push yourregistry/deislabs/ratify:yourtag
 ```
-3. Update [values.yaml](https://github.com/deislabs/ratify/blob/main/charts/ratify/values.yaml) to pull from your registry, when reusing image tag, setting pull policy to "Always" ensures we are pull the new changes
+
+#### Update [values.yaml](https://github.com/deislabs/ratify/blob/main/charts/ratify/values.yaml) to pull from your registry, when reusing image tag, setting pull policy to "Always" ensures we are pull the new changes
 ```json
 image:
   repository: yourregistry/deislabs/ratify
   tag: yourtag
   pullPolicy: Always
 ```
-4. Deploy from local helm chart
+
+### Deploy from local helm chart
+Deploy using one of the following deployments.
+
+**Option 1**
+Client and server auth disabled
```

Comment:
@dtzar should we consider adding this block to a separate TLS document or maybe an advanced setup document?

Reply:
Agree we should move this to a separate document and leave it disabled for now.